### PR TITLE
[DO NOT MERGE ] Update Angular 1.x articles

### PR DESCRIPTION
**Pull request by ValeriyRadchenko**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/angularjs/00-intro.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction
+description: test
+---
+
+
+
+<%= include('../../_includes/_signup') %>
```
Comment (chenkie) [inline articles/client-platforms/angularjs/00-intro.md]: we just made the decision to remove this yesterday. can we remove these _signup includes from the articles?

File: articles/client-platforms/angularjs/00-intro.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Prerequisites
+
+Before starting, it is important to ensure your application is set up correctly by following these steps:
+
+### 1. Create an Application
+
+<%= include('../../_includes/_new_app') %>_
+
+### 2. Configure Callback URLs
```
Comment (chenkie) [inline articles/client-platforms/angularjs/00-intro.md]: Let's make these headings h2's instead (##) and without numbers. So this one becomes `## Configure Callback URLs`

File: articles/client-platforms/angularjs/00-intro.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+A callback URL is a URL in your application where Auth0 redirects to after the user has authenticated. You can set the Callback URL by going to the [Application Settings](${uiAppSettingsURL}) section of your Auth0 dashboard and make sure that **Allowed Callback URLs** contains the following value:
+
+<pre><code>http://localhost:3000/</pre></code>
```
Comment (chenkie) [inline articles/client-platforms/angularjs/00-intro.md:30]: Can we add a comment that says the samples use `locahost:3000` and that this shouldbe placed as a callback URL? And some wording to reflect that if they are using a different host:port for their app, they need to put the appropriate one.

File: articles/client-platforms/angularjs/01-login.md [hunk 4]
```diff
@@ -1,118 +1,240 @@
...
-description: This tutorial will show you how to use the Auth0 with Angular 1.x applications.
+title: Login
+description: This tutorial will show you how to use the Auth0 Angular SDK to add authentication and authorization to your mobile app.
 ---
 
-<%= include('../../_includes/_package2', {
-  org: 'auth0-samples',
-  repo: 'auth0-angularjs-sample',
-  path: '00-Starter-Seed'
+<%= include('../../_includes/_package', {
```
Comment (chenkie) [inline articles/client-platforms/angularjs/01-login.md]: We've updated the packager to version 2. Can you take a look at how I've configured the new packager [here](https://github.com/auth0/docs/blob/master/articles/client-platforms/angularjs/01-login.md) and follow that pattern for all of them?

File: articles/client-platforms/angularjs/01-login.md [hunk 5]
```diff
@@ -1,118 +1,240 @@
...
-${include('../\_callback')}
+### 0. Create Auth0 service
 
-#### Create an Application Instance
+The best way to have authentication utilities available across the application is to use an injectable service.
+Notice the `authenticated` success callback which will store the `id_token` in the Local Storage. This value can be retrieved at a later stage.
 
-<%= include('../../_includes/_new_app') %>
+```js
+/* ===== components/auth/auth.service.js ===== */
```
Comment (chenkie) [inline articles/client-platforms/angularjs/01-login.md]: for code comment which point to the file path, let's remove the `=====` and also standardize how we do the comment.

For JavaScript files it should be:

`// components/auth/auth.service.js`

and for HTML it should be:

`<!-- components/home/home.template.html -->`

File: articles/client-platforms/angularjs/02-custom-login.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
+---
+title: Custom Login
+description: This tutorial will show you how to use the Auth0 Angular 1.x SDK to add authentication and authorization to your mobile app.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/02-custom-login.md]: all the descriptions should read "This tutorial demonstrates how to..." instead of "This tutorial will show you how to..."

File: articles/client-platforms/angularjs/02-custom-login.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+  pkgFilePath: '02-Custom-Login/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
+
+The previous step explained how you can log users into your application using the Lock Widget. You do not have to use Lock, and can instead create a custom login page and log the user in using a username and password they supply through a customer user interface.
+
+If you are using social logins, you can also launch the login screen for a particular social login provider directly from your Angular application.
+
+### 1. Update references
```
Comment (chenkie) [inline articles/client-platforms/angularjs/02-custom-login.md]: Make sure all headings have proper title casing: http://www.grammar-monster.com/lessons/capital_letters_title_case.htm

File: articles/client-platforms/angularjs/00-intro.md (hunk 1 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/00-intro.md]: Done

File: articles/client-platforms/angularjs/00-intro.md (hunk 2 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/00-intro.md]: Done

File: articles/client-platforms/angularjs/00-intro.md (hunk 3 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/00-intro.md:30]: Done

File: articles/client-platforms/angularjs/01-login.md (hunk 4 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/01-login.md]: Done

File: articles/client-platforms/angularjs/01-login.md (hunk 5 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/01-login.md]: Done

File: articles/client-platforms/angularjs/02-custom-login.md (hunk 6 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/02-custom-login.md]: Done

File: articles/client-platforms/angularjs/02-custom-login.md (hunk 7 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/02-custom-login.md]: Done

File: articles/client-platforms/angularjs/03-session-handling.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    ...
+})();
+
+```
+
+## Check Session
+
+To check if a user is authenticated, we can use `tokenNotExpired` from [angular-jwt](https://github.com/auth0/angular-jwt) which allows us to check whether the user's JWT is expired or not. Since JWT is a "stateless" manner of doing user authentication, the best way to know if the user should be regarded as authenticated on the front end is to know whether the token is unexpired.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/03-session-handling.md]: This should say "we can use `jwtHelper.isTokenExpired`" instead of `tokeNotExpired`

File: articles/client-platforms/angularjs/03-session-handling.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+To use this service, inject `authService` into your component `run` method and call `authService.checkAuthOnRefresh()`:
+
+```js
+// app.run.js
+
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/03-session-handling.md]: Ellipses on this line can be removed

File: articles/client-platforms/angularjs/01-login.md [hunk 10]
```diff
@@ -1,118 +1,237 @@
...
+description: This tutorial demonstrates how to use the Auth0 Angular SDK to add authentication and authorization to your mobile app.
 ---
 
 <%= include('../../_includes/_package2', {
   org: 'auth0-samples',
   repo: 'auth0-angularjs-sample',
-  path: '00-Starter-Seed'
+  path: '01-Login',
+  pkgFilePath: '01-Login/auth0.variables.js',
+  pkgType: 'replace'
```
Comment (chenkie) [inline articles/client-platforms/angularjs/01-login.md]: We need to change over to the new version of the packager for all the steps. Have a look at the [seed project instructions](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects) for the format, as well as the [current Angular doc](https://github.com/auth0/docs/blob/master/articles/client-platforms/angularjs/01-login.md) for an example.

File: articles/client-platforms/angularjs/01-login.md [hunk 11]
```diff
@@ -1,118 +1,237 @@
...
+    // can be accessed from the nav bar
+    $rootScope.authService = authService;
 
-      ...
+    // Register the authentication listener that is
+    // set up in auth.service.js
+    authService.registerAuthenticationListener();
 
-    });    
+    //Register the synchronous hash parser
```
Comment (chenkie) [inline articles/client-platforms/angularjs/01-login.md]: We should add a comment whenever we call `lock.interceptHash` that it is only required if using UI Router. We should also make note of that in the intro somewhere.

File: articles/client-platforms/angularjs/03-session-handling.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
+---
+title: Session Handling
+description: This tutorial demonstrates how to integrate Auth0 with angular to add session handling and logout to your web app.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/03-session-handling.md]: "with angular" => "with Angular 1.x"

Check the other descriptions for this too :)

File: articles/client-platforms/angularjs/03-session-handling.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+(function () {
+
+  ...
+
+  function authService($rootScope, lock, authManager, jwtHelper) {
+  
+	...
+
+    function checkAuthOnRefresh() {
```
Comment (chenkie) [inline articles/client-platforms/angularjs/03-session-handling.md]: This method is available on `authManager`. Why do we have it overridden here?

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 14]
```diff
@@ -0,0 +1,183 @@
+---
+title: User Profile
+description: This example shows how to display the user's profile
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: make the descriptions consistent between steps. "This example demonstrates how to..."

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-angularjs-sample',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-angularjs-sample',
+  pkgBranch: 'master',
+  pkgPath: '04-User-Profile',
+  pkgFilePath: '04-User-Profile/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
+<%= include('../../_includes/_signup') %>
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: Remove the signup include from all steps

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+
+You can access obtain a user's profile from the `getProfile()` method.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: Remove the word "access"

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 17]
```diff
@@ -0,0 +1,183 @@
...
+
+## Accessing user profile with `lock.getProfile()`
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: Accessing the User's Profile with `lock.getProfile()`

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 18]
```diff
@@ -0,0 +1,183 @@
...
+
+At any given time, you can call `getProfile` on `lock` passing in a token and callback function.
+
+```js
+/* ===== components/auth/auth.service.js ===== */
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: remove ellipses on this line

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 19]
```diff
@@ -0,0 +1,183 @@
...
+
+  function authService($rootScope, lock, authManager, jwtHelper, $q) {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md:24]: remove ellipses on this line

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 20]
```diff
@@ -0,0 +1,183 @@
...
+          }
+
+          localStorage.setItem('profile', JSON.stringify(profile));
+          deferredProfile.resolve(profile);
+        });
+
+      });
+    }
+	
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: remove ellipses on this line

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 21]
```diff
@@ -0,0 +1,183 @@
...
+	
+  }
+})();
+```
+
+As you can see we are storing the `profile` in Local storage in the success callback. Once that is done, all you need to do is to retrieve the profile from Localstorage at a later stage. An example can be found in the `HomeController`:
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: Remove "As you can see"

Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: "all you need to do" => "all that is needed is to"

Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md]: "Localstorage" => "local storage"

File: articles/client-platforms/angularjs/04-user-profile.md [hunk 22]
```diff
@@ -0,0 +1,183 @@
...
+    .module('app')
+    .controller('HomeController', HomeController);
+
+  HomeController.$inject = ['authService'];
+
+  function HomeController(authService) {
+
+    var vm = this;
+    vm.authService = authService;
+
```
Comment (chenkie) [inline articles/client-platforms/angularjs/04-user-profile.md:63]: remove extra line here

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 23]
```diff
@@ -0,0 +1,252 @@
...
+
+<%= include('../../_includes/_signup') %>
+
+<%= include('../../_includes/_linking_accounts') %>
+
+```js
+// components/auth/auth.service.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 24]
```diff
@@ -0,0 +1,252 @@
...
+
+  function authService($rootScope, lock, authManager, jwtHelper, $q, $http) {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 25]
```diff
@@ -0,0 +1,252 @@
...
+	
+    ...
+
+    return {
+	
+     ...
+	 
+      linkAccount: linkAccount,
+	  
+     ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 26]
```diff
@@ -0,0 +1,252 @@
...
+	 
+    }
+  }
+})();
+
+```
+
+Now that the second login is handled, you will need to actually do the linking.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: This should say "Now that the second user is authenticated, the accounts can be linked"

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 27]
```diff
@@ -0,0 +1,252 @@
...
+            });
+          
+          });
+
+    });  
+```
+
+This function posts to the API, passing the `link_with` parameter with the JWT value in the body. It then fetches the profile on success to check that the accounts are linked.
+
+Now to begin the link process, call the `linkAccount` method.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove the word "Now"

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 28]
```diff
@@ -0,0 +1,252 @@
...
+
+```js
+// components/home/home.controller.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 29]
```diff
@@ -0,0 +1,252 @@
...
+
+    function linkAccount() {
+      authService.linkAccount()
+        .then(function (profile) {
+          vm.profile = profile;
+          refreshIdentities();
+        })
+    }
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 30]
```diff
@@ -0,0 +1,252 @@
...
+
+If you fetch the profile after linking accounts, this same information will be available. 
+
+You can display this information and provide an **Unlink** button:
+
+```html
+<!-- www/components/home/home.html -->
+  ...
+	
+  <div class="identities-wrap">
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: See the latest code I pushed in the samples for an updated view

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 31]
```diff
@@ -0,0 +1,252 @@
...
+  ...
+```
+
+The user's primary identity can be filtered by putting in a function to refresh the identities.
+
+```js
+// components/home/home.controller.js
+(function () {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 32]
```diff
@@ -0,0 +1,252 @@
...
+  function HomeController($scope, authService) {
+
+   ...
+
+    function refreshIdentities() {
+      vm.profile.identities.shift();
+      vm.identities = vm.profile.identities;
+    }
+
+   ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 33]
```diff
@@ -0,0 +1,252 @@
...
+
+## Unlinking Accounts
+
+You can dissociate a linked account by calling the [unlink a user account](/api/management/v2#!/Users/delete_provider_by_user_id) endpoint using the primary `user_id`, and the `provider` and `user_id` of the identity to unlink.
+
+```js
+// components/auth/auth.service.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/05-linking-accounts.md [hunk 34]
```diff
@@ -0,0 +1,252 @@
...
+
+        });
+
+      return deferred.promise;
+
+    }
+
+    return {
+	
+      ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/05-linking-accounts.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/07-authorization.md [hunk 35]
```diff
@@ -0,0 +1,87 @@
...
+
+## Restrict Access to Secure Content
+
+To restrict secure content to users with a role of `admin`, subscribe to `$stateChangeStart` event.
+
+```js
+// components/auth/auth.service.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/07-authorization.md [hunk 36]
```diff
@@ -0,0 +1,87 @@
...
+
+  function authService($rootScope, lock, authManager, jwtHelper, $q) {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/07-authorization.md [hunk 37]
```diff
@@ -0,0 +1,87 @@
...
+<div class="jumbotron">
+  <h2 class="text-center"><img src="https://cdn.auth0.com/styleguide/1.0.0/img/badge.svg"></h2>
+  <h2 class="text-center">Admin</h2>
+  <div class="text-center">
+    <h2>You are viewing this because you are logged in and you have 'admin' role</h2>
+  </div>
+</div>
+```
+
+Every time when `$stateChangeStart` event fires we check if the user is an admin using a new `isAdmin` function added to the `authService`. This method checks if the `roles` attribute of `app_metadata` added by the rule contains `admin`.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: This should say "Every time the `$stateChangeStart`..."

File: articles/client-platforms/angularjs/07-authorization.md [hunk 38]
```diff
@@ -0,0 +1,87 @@
...
+
+
+```js
+// components/auth/auth.service.js
+(function () {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/07-authorization.md [hunk 39]
```diff
@@ -0,0 +1,87 @@
...
+
+  function authService($rootScope, lock, authManager, jwtHelper, $q) {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/07-authorization.md (hunk 37 above)
Comment (chenkie) [inline articles/client-platforms/angularjs/07-authorization.md]: Add a note after this paragraph in a blockquote

> **Note:** Users have no control over their own `app_metadata`, so there is no risk of a user modifying their own access level in Auth0. Keep in mind, however, that the payload of a JSON Web Token can be modified in debuggers such as [jwt.io](https://jwt.io). If a user does this, their JWT will be invalidated and become unusable for accessing server resources. They would, however, be able to access client side routes with a modified payload. Be sure to keep sensitive information out of the client side and rely on XHR requests for that information as this will ensure that resources are properly protected."

File: articles/client-platforms/angularjs/08-calling-apis.md [hunk 40]
```diff
@@ -0,0 +1,148 @@
...
+
+## JWT Interceptor
+
+To attach the user's JWT as an `Authorization` header, we could write a service that returns a token and attaches it to all HTTP requests. However, the **angular-jwt** library already provides this functionality. Ensure that **angular-jwt** is added to your app as a dependency and configure it with `jwtOptionsProvider` in the `config` block. A `tokenGetter` function needs to be provided to inform **angular-jwt** how it should be retrieving the user's JWT.
+
+```js
+// app.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/08-calling-apis.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/08-calling-apis.md [hunk 41]
```diff
@@ -0,0 +1,148 @@
...
+## Not Sending the JWT for Specific Requests
+
+This basic example will attach the JWT as an `Authorization` header to all requests. This may not be desired as some requests don't require authentication. You can choose not to send the JWT by specifying `skipAuthorization: true`.
+
+```js 
+// components/ping/ping.service.js
+
+(function () {
+
+  ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/08-calling-apis.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/08-calling-apis.md [hunk 42]
```diff
@@ -0,0 +1,148 @@
...
+
+## Avoiding Template Requests
+
+Remember that template requests via `ui-router` or `ng-route` are HTTP requests. This means that `Authorization` headers will be attached as well and that might not be needed. You may provide some configuration to avoid sending the JWT for template requests.
+
+```js
+// app.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/08-calling-apis.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/08-calling-apis.md [hunk 43]
```diff
@@ -0,0 +1,148 @@
...
+
+## Different Tokens Based on URLs
+
+If for any reason you would want to send different tokens based on different URLs, you can configure the `tokenGetter` to do so.
+
+```js
+// app.js
+(function () {
+
+	...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/08-calling-apis.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/09-mfa.md [hunk 44]
```diff
@@ -0,0 +1,26 @@
...
+  path: '09-MFA,
+  pkgFilePath: '09-MFA/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
+<%= include('../../_includes/_signup') %>
+
+<%= include('../_includes/_mfa-introduction') %>
+
+In this tutorial, you will learn how to enable MFA in the Angular 1.x application you created in the previous steps.
```
Comment (chenkie) [inline articles/client-platforms/angularjs/09-mfa.md]: Reword this to:

"This tutorial demonstrates how to enable..."

File: articles/client-platforms/angularjs/10-customizing-lock.md [hunk 45]
```diff
@@ -0,0 +1,87 @@
...
+## Theme Options
+
+You can set custom theme properties, such as a different logo or primary color, by adding a `theme` property with custom values:
+
+```js
+// app.js
+
+(function () {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/10-customizing-lock.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/10-customizing-lock.md [hunk 46]
```diff
@@ -0,0 +1,87 @@
...
+## Language Dictionary Specification
+
+You can also customize the text that `Lock` will display with the `languageDictionary` option parameter:
+
+```js
+// app.js
+
+(function () {
+
+    ...
```
Comment (chenkie) [inline articles/client-platforms/angularjs/10-customizing-lock.md]: Remove ellipses on this line

File: articles/client-platforms/angularjs/03-session-handling.md (hunk 13 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/03-session-handling.md]: because it didn't work properly when I was working on examples

File: articles/client-platforms/angularjs/03-session-handling.md (hunk 12 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/03-session-handling.md]: Done

File: articles/client-platforms/angularjs/03-session-handling.md (hunk 8 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/03-session-handling.md]: Done

File: articles/client-platforms/angularjs/03-session-handling.md (hunk 9 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/03-session-handling.md]: Done

File: articles/client-platforms/angularjs/01-login.md (hunk 10 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/01-login.md]: Done

File: articles/client-platforms/angularjs/01-login.md (hunk 11 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/01-login.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 14 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 15 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 16 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 17 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 18 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 19 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md:24]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 20 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 21 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md]: Done

File: articles/client-platforms/angularjs/04-user-profile.md (hunk 22 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/04-user-profile.md:63]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 23 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 24 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 25 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 26 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 27 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 28 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 29 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 30 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 31 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 32 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 33 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/05-linking-accounts.md (hunk 34 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/05-linking-accounts.md]: Done

File: articles/client-platforms/angularjs/07-authorization.md (hunk 35 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

File: articles/client-platforms/angularjs/07-authorization.md (hunk 36 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

File: articles/client-platforms/angularjs/07-authorization.md (hunk 37 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

File: articles/client-platforms/angularjs/07-authorization.md (hunk 38 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

File: articles/client-platforms/angularjs/07-authorization.md (hunk 39 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/07-authorization.md]: Done

File: articles/client-platforms/angularjs/08-calling-apis.md (hunk 40 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/08-calling-apis.md]: Done

File: articles/client-platforms/angularjs/08-calling-apis.md (hunk 41 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/08-calling-apis.md]: Done

File: articles/client-platforms/angularjs/08-calling-apis.md (hunk 42 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/08-calling-apis.md]: Done

File: articles/client-platforms/angularjs/08-calling-apis.md (hunk 43 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/08-calling-apis.md]: Done

File: articles/client-platforms/angularjs/09-mfa.md (hunk 44 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/09-mfa.md]: Done

File: articles/client-platforms/angularjs/10-customizing-lock.md (hunk 45 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/10-customizing-lock.md]: Done

File: articles/client-platforms/angularjs/10-customizing-lock.md (hunk 46 above)
Comment (ValeriyRadchenko) [reply articles/client-platforms/angularjs/10-customizing-lock.md]: Done

**Comment by chenkie** [comment]
Closing as I put this content into another PR to do some additional editing.
